### DISCUSSION TITLE
An Empire reference

### DISCUSSION THREAD

**Discussion by GitFernandoGalaz** [sampsapursiainen/zeffiro_interface#17] [Q&A] [answered]
How do you turn this on? o:

**Comment by GitFernandoGalaz** [comment, marked as answer]
https://en.wikipedia.org/wiki/File:AC_Cobra_-_50th_Annyversary_Cobra_Limited_Edition_CSX8000_(02).jpg

**Reply by sampsapursiainen** [reply]
https://www.youtube.com/watch?v=ELUl-cAtUIE
